bakery/checkers/checkers.go

```diff
@@ -235,3 +237,48 @@ func ClientIPAddrCaveat(addr net.IP) Caveat {
 func ErrorCaveatf(f string, a ...interface{}) Caveat {
 	return firstParty(CondError, fmt.Sprintf(f, a...))
 }
+
+// AllowCaveat returns a caveat that will deny attempts to use the
+// macaroon to perform any operation other than those listed.
+func AllowCaveat(op ...string) Caveat {
+	return firstParty(CondAllow, strings.Join(op, " "))
```

Comment:
Perhaps check that none of the operations contains a " " character and return an error caveat if they do?